### PR TITLE
Add mandatory OG tag for video pages

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## What does this change?

Adds a `meta` tag to appease the god of open graph protocols.

Screenshot 2019-07-11 at 11 00 08

## What is the value of this and can you measure success?

Our social cache clearing lambda is failing many times with the following error:

> Object at URL "http://www.theguardian.com/world/video/2019/jul/08/scientists-discover-snowball-the-cockatoo-has-16-distinct-dance-moves-video" of type "article" is invalid because a required property "og:video:url" of type "url" was not provided.

### Tested

- [x] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: common/app/model/content.scala [hunk 1]
```diff
@@ -632,7 +632,13 @@ object Video {
       "videoDuration" -> elements.videos.find(_.properties.isMain).map{ v => JsNumber(v.videos.duration)}.getOrElse(JsNull)) ++ AtomProperties(content.atoms)
 
 
-    val optionalOpengraphProperties = if(content.metadata.webUrl.startsWith("https://")) Map("og:video:secure_url" -> content.metadata.webUrl) else Nil
+    val optionalOpengraphProperties =
+      if(content.metadata.webUrl.startsWith("https://"))
```
Comment (nicl) [inline common/app/model/content.scala]: Is this an example where the else branch should never be hit?

**Review by nicl — APPROVED** [review]

File: common/app/model/content.scala (hunk 1 above)
Comment (regiskuckaertz) [reply common/app/model/content.scala]: I'm not sure... let's ask @akash1810 ?

Comment (akash1810) [reply common/app/model/content.scala]: Yes there are, sadly. We should to re-write them in Composer as there is no reason for them not to be `https`.

**Comment by regiskuckaertz** [comment]
[Fix social clearing](https://trello.com/c/Q5nklxbD/759-fix-social-clearing)

**Pull request merged**